vllm/engine/arg_utils.py

```diff
@@ -228,6 +245,7 @@ class EngineArgs:
     kv_cache_dtype: CacheDType = CacheConfig.cache_dtype
     seed: Optional[int] = None
     max_model_len: Optional[int] = None
+    cuda_graph_sizes: Optional[Union[int, List[int]]] = 512
```

Comment:
Can we try and keep default values in `config.py` only? This makes changing them in future a simpler task

```suggestion
    cuda_graph_sizes: Optional[Union[int, List[int]]] = SchedulerConfig.cuda_graph_sizes
```